package.json

```diff
@@ -14,7 +14,7 @@
     "build": "tsc",
     "build:ci": "ncc build src/index.ts -o dist && mv ./dist/index.js ./bin",
     "start": "probot run ./lib/main.js",
-    "start:watch": "nodemon --exec 'yarn start'",
+    "start:watch": "copy-folder-util src/assets/images lib/assets/images && nodemon --exec 'yarn start'",
```

Comment:
This seems really unnecessary to add this dependency. What's the issue with `cp -r` or `rsync`

Reply:
I'm thinking about cross platform support, whichever command or dependency works
@pavlovcik

Reply:
I thought that `cp` works for windows lol